### DISCUSSION TITLE
How to train graph classification on IMDB-BINARY?

### DISCUSSION THREAD

**Discussion by AdarshMJ** [pyg-team/pytorch_geometric#6933] [Q&A] [answered]
Hi @rusty1s, @EdisonLeeeee 
The IMDB, REDDIT and COLLAB datasets have 0 node features. How to perform graph classification on this?
`Data(edge_index=[2, 146], y=[1], num_nodes=20)`
I keep getting this error - `AttributeError: 'NoneType' object has no attribute 'size' `

**Comment by rusty1s** [comment, marked as answer]
For these, you need to create node features on your own. The most common one is to use node degree as features, e.g., via `transforms.OneHotDegree`.

**Reply by AdarshMJ** [reply]
Yes.. got it. Thank you!
